watcher/watcher.go

```diff
@@ -746,26 +751,26 @@ func (w *Watcher) RemoveDeadRooms() error {
 		)
 
 		if timedout {
-			logger.WithError(err).Error("timeout replacing pods on RemoveDeadRooms")
+			return errors.New("timeout replacing pods on RemoveDeadRooms")
 		}
 
 		if err != nil {
-			logger.WithError(err).Error("replacing pods returned error on RemoveDeadRooms")
+			return fmt.Errorf("replacing pods returned error on RemoveDeadRooms: %s", err)
 		}
 
-		if timeoutErr == false && err == nil {
-			l := logger.WithFields(logrus.Fields{
+		if len(roomsNoPingSince) > 0 {
+			logger.WithFields(logrus.Fields{
 				"rooms": fmt.Sprintf("%v", roomsNoPingSince),
-			})
-			l.Info("successfully deleted rooms that were not pinging")
+			}).Info("successfully deleted rooms that were not pinging")
```

Comment:
Shouldn't this become a warning, like the log for zombie rooms? Same question applies for rooms that expired occupied timeout.